src/metadata.rs

```diff
@@ -63,6 +64,7 @@ pub fn fetch_metadata(provider: &str) -> errors::Result<Box<dyn providers::Metad
         "ibmcloud" => box_result!(IBMGen2Provider::try_new()?),
         // IBM Cloud - Classic infrastructure.
         "ibmcloud-classic" => box_result!(IBMClassicProvider::try_new()?),
+        "one" => box_result!(ContextDrive::try_new()?),
```

Comment:
Is this an arbitrary ID or is there prior usage of this specific value anywhere in the CoreOS world?

Reply:
I'm not aware of "one" being used as the short moniker for OpenNebula within the CoreOS world, but it is the predominantly used short reference name for OpenNebula in other areas, like `oneadmin` as the default login user, or the command line tools `onevm`, etc. I chose the short name here to match the other used short references like "gcp" and "aws", but I'll leave the final decision up to you.

Reply:
Thanks for the feedback. I brought this up yesterday in the [weekly FCOS meeting](https://meetbot.fedoraproject.org/teams/fedora_coreos_meeting/fedora_coreos_meeting.2020-08-12-16.30.html) and the broad consensus was to use `opennebula` as the ID for this platform.

Part of the rationale is that `one` is too easy to cause confusion when casually mentioned in isolation, outside of the specific OpenNebula context.

We can freely keep the `ONE_` prefix for attributes, I think the shorthand makes sense there.

Reply:
> We can freely keep the `ONE_` prefix for attributes, I think the shorthand makes sense there.

Shouldn't we keep the prefix consistent with the platform ID?  The only case where we currently diverge is the `-configdrive` and `-metadata` providers.